### PR TITLE
[CP-857] Refactor test_messages.py and test_send_sms.py

### PR DESCRIPTION
**Pull request by pawel-mudita**
Refactored sms part of messages endpoint tests.
Rewritten with use of messages EP API from harness.

### CODE REVIEW

File: test/pytest/service-desktop/test_messages.py [hunk 1]
```diff
@@ -1,203 +1,376 @@
...
+    initial_number_of_messages_records = received_messages_records_count
+
+    message_number_1 = "123456789"
+    message_1_body = "Hello, how are you?"
+    message_2_body = "Are you there?"
+
+    message_number_2 = "223456789"
+    message_3_body = "What's up?"
+    message_4_body = "Did you checked your email recently?"
+
```
Comment (pholat) [inline test/pytest/service-desktop/test_messages.py:172]: instead of copying why not create for loop? same below

**Review by wojtekrzepecki — APPROVED** [review]

File: test/pytest/service-desktop/test_messages.py (hunk 1 above)
Comment (pawel-mudita) [reply test/pytest/service-desktop/test_messages.py]: Added some loops

File: test/pytest/service-desktop/test_messages.py [hunk 2]
```diff
@@ -1,203 +1,267 @@
...
+from harness.request import TransactionError
+from harness.api.messages import GetMessagesCount, GetMessagesWithOffsetAndLimit, GetMessageById, \
+    GetMessagesByThreadIdWithOffsetAndLimit, AddMessage, DeleteMessageById
+
+
+class MessagesTester:
+    def __init__(self, harness):
+        self.harness = harness
+
+    def get_messages_count(self):
```
Comment (pholat) [inline test/pytest/service-desktop/test_messages.py:13]: I have djavu - there were tests which had the very same functions last year?

File: test/pytest/service-desktop/test_send_sms.py [hunk 3]
```diff
@@ -32,13 +31,8 @@ def test_send_sms_too_long(harness, phone_number):
...
-
-    sms_text = sms_text + 'a'
-    body = {"category": "message", "number": str(phone_number), "messageBody": sms_text}
-    ret = harness.endpoint_request("messages", "post", body)
-    assert ret["status"] == status["BadRequest"]
-
+    ret = AddMessage(str(phone_number), sms_text).run(harness)
 
+    sms_text = sms_text + "a"
+    with pytest.raises(TransactionError, match=r".*" + str(Status.BadRequest.value) + ".*"):
```
Comment (pholat) [inline test/pytest/service-desktop/test_send_sms.py:37]: why should it rise TransactionError -> BadRequest?

Comment (pawel-mudita) [reply test/pytest/service-desktop/test_send_sms.py:37]: It is an expected error, when we try to send a too long message.

File: test/pytest/service-desktop/test_messages.py (hunk 2 above)
Comment (pawel-mudita) [reply test/pytest/service-desktop/test_messages.py:13]: I guess you are talking about functions of messages API from harness tests repo.

**Review by niepiekm — APPROVED** [review]

**Pull request merged**